jepsen/src/jepsen/control/k8s.clj

```diff
@@ -0,0 +1,111 @@
+(ns jepsen.control.k8s
```

Comment:
I believe they're used interchangealby, so it can be expected that someone who's looking for "kubernetes" will also be looking fof k-eight-s or "k8s". I'm open here, too.